### PR TITLE
Add Hubot and Mona GitHub fonts

### PR DESCRIPTION
**Pull request by drupol**
New open-source Github fonts available: https://github.com/mona-sans
 
This PR:

- Add the Github Mona Sans font
- Add the Github Hubot Sans font

This is the first time I add a font in Nix, please review it

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/hubot-sans/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "hubot-sans";
+  version = "1.0";
+
+  src = fetchFromGitHub {
```
Comment (AndersonTorres) [inline pkgs/data/fonts/hubot-sans/default.nix:7]: I believe the procedure to install fonts is a bit different, but I need to look at the other expressions.

File: pkgs/data/fonts/hubot-sans/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub }:
```
Comment (AndersonTorres) [inline pkgs/data/fonts/hubot-sans/default.nix:1]: If there is no C compilation, use stdenvNoCC:

```suggestion
{ lib
, stdenvNoCC
, fetchFromGitHub 
}:
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/data/fonts/hubot-sans/default.nix (hunk 2 above)
Comment (drupol) [reply pkgs/data/fonts/hubot-sans/default.nix]: Very good, I didn't know that thanks :)

File: pkgs/data/fonts/hubot-sans/default.nix [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    repo = "hubot-sans";
+    sha256 = "GOql+V5TH4b3TmhlgnKcx3jzUAO2jm4HRJRNzdIKxgg=";
+  };
+
+  installPhase = ''
+    install -m644 --target $out/share/fonts/truetype/hubot-sans -D $src/dist/hubot-sans.ttf
+  '';
+
+  meta = {
+    description = "Hubot Sans, a variable font from GitHub";
```
[the remark this answers is not in the document]
Comment (drupol) [reply pkgs/data/fonts/hubot-sans/default.nix]: Thanks, I removed the reference to the name.

File: pkgs/data/fonts/hubot-sans/default.nix (hunk 1 above)
Comment (drupol) [reply pkgs/data/fonts/hubot-sans/default.nix:10]: I took some example from Inconsolata: https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/inconsolata/default.nix

Comment (AndersonTorres) [reply pkgs/data/fonts/hubot-sans/default.nix:10]: I will not complain, however I prefer this.

https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/data/fonts/b612/default.nix

Comment (drupol) [reply pkgs/data/fonts/hubot-sans/default.nix:10]: I tried, but for some reason I couldn't make it working. Had a conversation about that with Dee on Matrix:

image

Comment (drupol) [reply pkgs/data/fonts/hubot-sans/default.nix:10]: Feel free to let me know what to do to fix and I'll do the PR.

Comment (drupol) [reply pkgs/data/fonts/hubot-sans/default.nix:10]: @DeeUnderscore ^^

Comment (drupol) [reply pkgs/data/fonts/hubot-sans/default.nix:10]: Same issue with `b612` font: https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/data/fonts/b612/default.nix#L6

**Pull request merged**